adapters/lora/src/main/java/org/eclipse/hono/adapter/lora/LoraProtocolAdapter.java

```diff
@@ -269,6 +269,17 @@ void handleProviderRoute(final HttpContext ctx, final LoraProvider provider) {
                 currentSpan.finish();
                 // discard the message but return 202 to not cause errors on the LoRa provider side
                 handle202(ctx.getRoutingContext());
+
+                //TODO: add metrics
+                metrics.reportTelemetry(
```

Comment:
i'm in doubt whether this is the best spot to report the metrics here. If anyone has a better option, I'm all ears